pkg/util/timeutil/time.go

```diff
@@ -0,0 +1,16 @@
+package timeutil
+
+import "time"
+
+const (
+	nanosecondsInMillisecond = int64(time.Millisecond / time.Nanosecond)
+)
+
+// TimeFromMillis is a helper to turn milliseconds -> time.Time
```

Comment:
We already have similar functions in `pkg/util/time.go`, can we use that package instead of a new file?

Reply:
Definitely! I totally missed it.